ui/react-ui/src/enums/index.ts

```diff
@@ -0,0 +1,76 @@
+export enum TERMINAL_MESSAGE_TYPE {
+  PING = 'PING',
+  TERMINAL_INIT = 'TERMINAL_INIT',
+  TERMINAL_DATA = 'TERMINAL_DATA',
+  TERMINAL_SHARE = 'TERMINAL_SHARE',
+  TERMINAL_RESIZE = 'TERMINAL_RESIZE',
+  TERMINAL_K8S_DATA = 'TERMINAL_K8S_DATA',
+  TERMINAL_K8S_RESIZE = 'TERMINAL_K8S_RESIZE',
+  TERMINAL_SHARE_USER_REMOVE = 'TERMINAL_SHARE_USER_REMOVE',
+  TERMINAL_GET_SHARE_USER = 'TERMINAL_GET_SHARE_USER'
+}
+
+export enum SEND_LUNA_MESSAGE_TYPE {
+  PING = 'PING',
+  CLOSE = 'CLOSE',
+  PONG = 'PONG',
+  CONNECT = 'CONNECT',
+  TERMINAL_ERROR = 'TERMINAL_ERROR',
+  MESSAGE_NOTIFY = 'MESSAGE_NOTIFY'
+}
+
+export enum ZMODEM_ACTION_TYPE {
+  ZMODEM_START = 'ZMODEM_START',
+  ZMODEM_END = 'ZMODEM_END'
+}
+
+export enum MESSAGE_TYPE {
+  PING = 'PING',
+  CLOSE = 'CLOSE',
+  ERROR = 'ERROR',
+  CONNECT = 'CONNECT',
+  TERMINAL_SHARE = 'TERMINAL_SHARE',
+  TERMINAL_ERROR = 'TERMINAL_ERROR',
+  MESSAGE_NOTIFY = 'MESSAGE_NOTIFY',
+  TERMINAL_ACTION = 'TERMINAL_ACTION',
+  TERMINAL_SESSION = 'TERMINAL_SESSION',
+  TERMINAL_PERM_VALID = 'TERMINAL_PERM_VALID',
+  TERMINAL_SHARE_JOIN = 'TERMINAL_SHARE_JOIN',
+  TERMINAL_SHARE_LEAVE = 'TERMINAL_SHARE_LEAVE',
+  TERMINAL_PERM_EXPIRED = 'TERMINAL_PERM_EXPIRED',
+  TERMINAL_SESSION_PAUSE = 'TERMINAL_SESSION_PAUSE',
+  TERMINAL_SESSION_RESUME = 'TERMINAL_SESSION_RESUME',
+  TERMINAL_GET_SHARE_USER = 'TERMINAL_GET_SHARE_USER',
+  TERMINAL_SHARE_USER_REMOVE = 'TERMINAL_SHARE_USER_REMOVE'
+}
+
+export enum WINDOW_MESSAGE_TYPE {
+  PING = 'PING',
+  PONG = 'PONG',
+  CMD = 'CMD',
+  FOCUS = 'FOCUS',
+  OPEN = 'OPEN',
+  FILE = 'FILE',
+  CREATE_FILE_CONNECT_TOKEN = 'CREATE_FILE_CONNECT_TOKEN'
+}
+
+export enum SFTP_CMD {
+  RM = 'rm',
+  LIST = 'list',
+  MKDIR = 'mkdir',
+  MKFILE = 'mkfile',
+  RENAME = 'rename',
+  UPLOAD = 'upload',
+  DOWNLOAD = 'download'
+}
+
+export enum FILE_MANAGE_MESSAGE_TYPE {
+  CONNECT = 'CONNECT',
+  CLOSE = 'CLOSE',
+  ERROR = 'ERROR',
+  PING = 'PING',
+  PONG = 'PONG',
+  CLOSED = 'closed',
+  SFTP_DATA = 'SFTP_DATA',
+  SFTP_BINARY = 'SFTP_BINARY'
+}
```

Comment:
The provided code does not have any immediate apparent errors or issues. It includes multiple enumeration types, enums, and message-type strings like `PING`, `.close()`. The coding style is consistent but could be slightly cleaner in some cases. However, no major issues are observed based on the current knowledge cutoff available until 2021-01-30.

If you need assistance with specific features of these elements (like validating input), more context, clarifications or further debugging guidance please let me know!